src/scripts/run_certbot.sh

```diff
@@ -92,16 +92,16 @@ get_certificate() {
     certbot certonly \
         --agree-tos --keep -n --text \
         --preferred-challenges ${challenge_type} \
```

Comment:
I do not understand how shellcheck interprets this, because it thinks `${authenticator}` should be in quotes, but not `${challenge_type}`?

Furthermore, the other variables here cannot be quoted, since I use the fact that they are expanded into the flag `--force` and not the string `"--force"` for example. 
This might not be 100% safe, but all those are variables are controlled by us, so I think it is OK.